test/dlc_tests/container_tests/bin/testSmdebug

```diff
@@ -80,23 +80,9 @@ elif [[ $FRAMEWORK == "mxnet" ]]; then
         exit $exit_code
     fi
 else
-    TENSORFLOW_DATASETS_VERSION="4.0.1"
-    if [[ $FRAMEWORK == "tensorflow2" ]]; then
-        tf_version=$(python -c "import tensorflow as tf; print(tf.__version__)")
-        if [[ "$tf_version" == *"2.0."* ]]; then
-            TENSORFLOW_DATASETS_VERSION="3.2.1"
-        fi
-    fi
-    # pip install tensorflow-datasets==${TENSORFLOW_DATASETS_VERSION} "absl-py>=0.9,<0.11"
     for test_file in tests/${FRAMEWORK}/*
     do
-        
-        if [[ "$tf_version" == *"2.7."* && "$test_file" == *"test_keras_mirrored.py" ]]; then
```

Comment:
Is this removal backward compatible? In other words, if I run tests on TF2.7, would they succeed?

Reply:
no, tensorflow-datasets is already installed in the image

Reply:
Wait, by removing this check to skip tests on 2.7, if we re-run tests on TF 2.7, they will now fail. These lines should be left. Also, these tests no longer fail in 2.8 then??

Reply:
> no, tensorflow-datasets is already installed in the image

Since we're still supporting TF 2.7 images, there's a possibility that we may need to release new patches. If TF 2.7 images can pass this test now (no flakiness), we can safely remove these lines. Otherwise, we would still need to keep it.
```
if [[ "$tf_version" == *"2.7."* && "$test_file" == *"test_keras_mirrored.py" ]]; then
```

Reply:
no TF 2.7 cannot consistently pass this test (TF2.8 can)